src/components/VList/VListGroup.js

```diff
@@ -107,6 +107,13 @@ export default {
 
       this.isActive = !this.isActive
     },
+    onKeyDown: function onKeyDown (e) {
+      e.stopPropagation()
```

Comment:
Do we really want to do this? Probably better to check the target instead.

Reply:
Since this is bound directly to the list group header, won't that always be the target?

I'm probably going to end up removing it since I added it on autopilot and I can't think of any reason we'll need it.

Reply:
I think it's there to prevent toggling parents? It should be checked with `e.target === this.$el` to make sure it's not actually from a child element.

Reply:
added the check in